fe/fe-core/src/main/java/com/starrocks/statistic/StatisticSQLBuilder.java

```diff
@@ -51,8 +51,9 @@ public class StatisticSQLBuilder {
                     + " GROUP BY partition_id";
 
     private static final String QUERY_PARTITION_STATISTIC_TEMPLATE =
-            "SELECT cast(" + STATISTIC_PARTITION_VERSION + " as INT), " +
-                    " `partition_id`, `column_name`, hll_cardinality(hll_union(`ndv`)) as distinct_count"
+            "SELECT cast(" + STATISTIC_PARTITION_VERSION_V2 + " as INT), " +
+                    " `partition_id`, `column_name`, hll_cardinality(hll_union(`ndv`)) as distinct_count,"
```

Comment:
Why not use `hll_cardinality(any_value(`ndv`))`?

Reply:
I will modify it later